cpmonitor/models.py

```diff
@@ -39,13 +45,22 @@ class Meta:
         "Zieljahr Klimaneutralität", blank=True, null=True, help_text="z.B. 2035"
     )
 
-    introduction = models.TextField(
-        "Einleitung",
+    teaser = models.CharField(
+        "Teaser",
+        max_length=200,
         blank=True,
         help_text="""
-            <p>Eine kurze Beschreibung der Situation in der Kommune.</p>
-            <p>Kann in einer Übersicht aller Kommunen dargestellt werden.</p>
-            <p>Deswegen möglichst kurz und ohne Formatierungen. Fett, Kursiv, Links, etc. sind okay.</p>
+            <p>Eine kurze Beschreibung der Situation in der Kommune. Maximal 200 Zeichen. Keine Formatierungen.</p>
+            <p>Kann in einer Übersicht aller Kommunen oder als Vorschau eines Links dargestellt werden.</p>
```

Comment:
I think it would be more precise to say: "Wird in der Übersicht aller Kommunen und auf der Startseite der Kommune dargestellt. Kann auch als Vorschau eines Links dargestellt werden."

Reply:
to this and the next comment: These might be better - i am not sure. Much more important, that we have the stuff merged, and I lack the time to put in these corrections now. Help text can and should be reviewed by users.